### PR TITLE
make very consume tokens

### PR DESCRIPTION
**Pull request by AnEmortalKid**
makes `very` consume tokens as it parses.

Also adds tests for some weird edge cases that i hadn't seen.

### CODE REVIEW

File: lib/parser.js [hunk 1]
```diff
@@ -955,42 +956,51 @@ module.exports = function parse (line) {
...
-            // shift tokens
-            var anonKeys = keys.slice(3);
-            var anonStatement = handleLambda(anonKeys);
-            statement += anonStatement;
-            return statement;
-        }
-        if (keys.length > 4) {
-            var recurse = '';
-            for (var i = 3; i < keys.length; i++) {
-                if (keys[i].substr(-1) === ',' && keys[i].charAt(keys[i].length - 2) !== '}') keys[i] = keys[i].slice(0, -1);
```
Comment (AnEmortalKid) [inline lib/parser.js]: not entirely sure what this check was about....... probably a copy from how plz would add `,` to each token so splitting `{ a: b }` would break as `{`, `a:`, `b`, `}` . Couldn't think of a scenario for very that would trigger this tho

File: test/spec/var/very-is-raw-json/expect.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+var x = {
+    foo: bar,
+    baz: qwz
+}
```
Comment (vpzomtrrfrt) [inline test/spec/var/very-is-raw-json/expect.js]: I'd expect there to be a semicolon here

Comment (AnEmortalKid) [reply test/spec/var/very-is-raw-json/expect.js]: ah, that simplifies things then. I wasn't sure if that was required syntax or not ! i'll update it. Node seems to let me not have it:
```
$ node
> var x = {
... foo: 'bar',
... baz: 'qwz'
... }
undefined
> x
{ foo: 'bar', baz: 'qwz' }

```

But if it doesn't hurt to append `;` at the end, that simplifies some logic.
